Some errors occur when compiling typescript.

I can't understand what affect by this.
But we should remove these logs, because it's implicit bugs.

The list of causes is below.

- [ ] parameter of Overlay#constractor 'selector'  is mismatch between string and HTMLElements.
- [ ] missing 'let' on declear iterator value.
and so on...

```
[21:55:46] Starting 'typescript:compileToSingleFiles'...
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(35,63): error TS2345: Argument of type 'HTMLElement' is not assignable to parameter of type 'string'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(49,26): error TS2304: Cannot find name 'i'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(49,33): error TS2304: Cannot find name 'i'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(49,49): error TS2304: Cannot find name 'i'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(50,78): error TS2304: Cannot find name 'i'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(54,63): error TS2345: Argument of type 'HTMLElement' is not assignable to parameter of type 'string'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts(54,91): error TS2304: Cannot find name 'i'.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-maxlength.jquery.ts(24,41): error TS2346: Supplied parameters do not match any signature of call target.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-radiobox.ts(7,59): error TS7006: Parameter 'event' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-radiobox.ts(11,31): error TS7006: Parameter 'el' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-table.jquery.ts(21,35): error TS7006: Parameter 'renderIds' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-table.jquery.ts(21,46): error TS7006: Parameter 'disableRenderIds' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-table.jquery.ts(21,64): error TS7006: Parameter 'columnNumber' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-treebox.ts(13,62): error TS7006: Parameter 'dropdown' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-treebox.ts(21,54): error TS7006: Parameter 'treeBoxId' implicitly has an 'any' type.
../src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-treebox.ts(25,62): error TS7006: Parameter 'dropdown' implicitly has an 'any' type.
[21:55:48] TypeScript: 16 semantic errors
```

--- a/components/src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts
+++ b/components/src/main/resources/META-INF/resources/butterfaces-ts/butterfaces-ajax.ts
@@ -46,7 +46,7 @@ namespace ButterFaces {
                 case "success": // After update of HTML DOM based on ajax response..
                     // console.log('ajax request success');
 
-                    for (i = 0; i < ids.length; i++) {
+                    for (let i = 0; i < ids.length; i++) {
                         let $elementToEmable = $(document.getElementById(ids[i]));
 
                         if ($elementToEmable.length !== 0) {

